app/views/prosecution_cases/cd_api/_overall_defendants.html.haml

```diff
@@ -16,4 +16,4 @@
         %td.govuk-table__cell
           = l(overall_defendant.date_of_birth&.to_date)
         %td.govuk-table__cell
-          = overall_defendant.maat_reference || t('search.result.defendant.unlinked')
+          = overall_defendant.maat_reference.first != "Z" ?  overall_defendant.maat_reference : t('search.result.defendant.unlinked')
```

Comment:
Would the below not work? The logic sits in the decorator for this already.
```suggestion
          = overall_defendant.linked? ?  overall_defendant.maat_reference : t('search.result.defendant.unlinked')
```

Reply:
Ahhh yeah - this makes sense to use the decorator method. Tested locally and its still works as should. will commit suggestion